### PR TITLE
Retrieve data from backend

### PR DESCRIPTION
**Pull request by SYM1000**
In this PR code for getting the user's current open tab URL, getting notes from DB based on that URL and displaying the new notes to the user has been added :)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -74,3 +74,44 @@ function getProfileURL() {
     }
   });
 }
+
+// Create a new note and render it
```
Comment (rjkroege) [inline main.js:78]: You should have a . at the end of the sentence.

File: popup.js [hunk 2]
```diff
@@ -135,4 +135,7 @@ window.onload = function () {
     };
     // Get users profile image URL
     getProfileURL();
+    
+    // Display data for current page
```
Comment (rjkroege) [inline popup.js:139]: Ditto. Add period.

File: popup.html [hunk 3]
```diff
@@ -67,7 +69,7 @@
     </div>
 
     <!-- Created notes are inserted into this container in a template. -->
-    <div class="notes-container">
+    <div class="notes-container" id="notes-container">
```
Comment (sammurraytuesta) [inline popup.html:72]: Place id before class for consistency.

**Review by sammurraytuesta — CHANGES_REQUESTED** [review]
Looks good overall! Great work!

**Review by rjkroege — APPROVED** [review]

**Pull request merged**